backend/director/db/postgres/db.py

```
@@ -210,6 +212,82 @@ def delete_session(self, session_id: str) -> bool:
+        """Make a session public or private."""
+        try:
+            query = """
+                UPDATE sessions 
+                SET is_public = %s, updated_at = %s
+                WHERE session_id = %s
+            """
+            import time
+            current_time = int(time.time() * 1000)
+            self.cursor.execute(query, (is_public, current_time, session_id))
+            self.conn.commit()
+            return self.cursor.rowcount > 0
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)